kdf.c

```diff
@@ -109,8 +89,12 @@ void kdf_decryption_key(const char *username, const char *password, int iteratio
 		iterations = 1;
 
 	if (iterations == 1)
```

Comment:
Same comment as the other one regarding the definition of a macro with the minimum number of iterations.